web-app/js/portal/common/helpers.js

```diff
@@ -316,3 +316,8 @@ function getPortalBase(pathname) {
 function normaliseLongitude(longitude) {
     return (longitude + 540) % 360 - 180;
 };
+
+// Remove all child nodes of HTML elements from the DOM
+function clearContents(jQuerySelector) {
```

Comment:
It is so usable, I decided to stick it in helpers.js. There are so many convoluted ways to do this in Ext land, that are (or used to be) in the code, but because they are convoluted, they are hard to find. I think this is a great method to have.